test/template/test_flextemplate.py

```diff
@@ -192,7 +192,6 @@ def test_flextemplate_rotation(tmp_path):
             "type": "C39",
             "x1": 80,
             "y1": 10,
-            "x2": 70,
```

Comment:
Why changing those values?
It'b better if the exeact same PDF file could is generated BEFORE & AFTER this PR, to ensure backward-compatibility

Reply:
Ok then, good job